kunta-api-server/src/main/java/fi/otavanopisto/kuntaapi/server/rest/OrganizationsApiImpl.java

```diff
@@ -122,8 +135,19 @@ public Response deleteService(String organizationId, String serviceId) {
   }
   
   @Override
-  public Response listServiceElectornicChannels(String organizationId, String serviceId) {
-    return createNotImplemented("Not implemented");
+  public Response listServiceElectornicChannels(String organizationIdParam, String serviceIdParam) {
+    OrganizationId organizationId  = new OrganizationId(KuntaApiConsts.IDENTIFIER_NAME, organizationIdParam);
+    ServiceId serviceId = new ServiceId(KuntaApiConsts.IDENTIFIER_NAME, serviceIdParam);
+    
+    // TODO: Merge provider results
+    
+    List<ServiceElectronicChannel> result = new ArrayList<>();
+    for (ServiceChannelProvider serviceChannelProvider : getServiceChannelProviders()) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename "serviceChannelProvider" which hides the field declared at line 60. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AHiddenFieldCheck)